web/spec/supabase.yml

```diff
@@ -909,7 +909,7 @@ pages:
             .from('cities')
```

Comment:
looks like this should be `.from('countries')`

Reply:
You are right. Fixed and merging.